src/app/components/MediaLoader/utils/urlConstructors.ts

```diff
@@ -0,0 +1,61 @@
+import { getEnvConfig } from '#app/lib/utilities/getEnvConfig';
+import isLive from '#app/lib/utilities/isLive';
+import parseAvRoute from '#app/routes/utils/parseAvRoute';
+
+const LIVE_BASE_URL = 'https://www.bbc.com';
+const TEST_BASE_URL = 'https://www.test.bbc.com';
+
+const LIVE_AMP_URL = 'https://web-cdn.api.bbci.co.uk';
+const TEST_AMP_URL = 'https://web-cdn.test.api.bbci.co.uk';
+const DEV_AMP_URL = TEST_AMP_URL;
+
+const isDev = () => getEnvConfig().SIMORGH_APP_ENV === 'local';
+
+const getAmpBaseUrl = () => {
+  switch (true) {
+    case isLive():
+      return LIVE_AMP_URL;
+    case isDev():
+      return DEV_AMP_URL;
+    default:
+      return TEST_AMP_URL;
+  }
+};
```

Comment:
Could we simplify this - looks like we don't need a switch/case statement, but a simple if/ternary statement. We can get rid of all references to dev & just keep test & live. 

```suggestion

const getAmpBaseUrl = () => {
  return isLive() ? LIVE_AMP_URL : TEST_AMP_URL;
};
```

Reply:
In fact, we could delete this function, and simply do this on line 34